pkg/genlib/generator_interface.go

```diff
@@ -934,10 +1114,11 @@ func bindDoubleWithReturn(fieldCfg ConfigField, field Field, fieldMap map[string
 		return nil
 	}
 
+	var emitF emitF
```

Comment:
What is the reason this was moved?

Reply:
no specific reason, it happened while working on the file. I will revert :)